Magento_InventoryInStorePickupFrontend/templates/success.phtml

```diff
@@ -0,0 +1,40 @@
+<div class="success-page__order-confirmation">
+    <?php if ($block->getOrderId()) : ?>
+        <?php if ($block->getCanViewOrder()) : ?>
+            <p>
+                <?= $block->escapeHtml(__('Your order number is: %1.', sprintf('<a href="%s" class="order-number"><strong>%s</strong></a>', $block->escapeUrl($block->getViewOrderUrl()), $block->getOrderId())), ['a', 'strong']) ?>
+            </p>
+        <?php else : ?>
+            <p>
+                <?= $block->escapeHtml(__('Your order # is: <span>%1</span>.', $block->getOrderId()), ['span']) ?>
+            </p>
+        <?php endif; ?>
+        <?php if ($block->getIsOrderStorePickup()->execute()) : ?>
+            <p>
+                <?= $block->escapeHtml(__('We\'ll email you an order and pick-up confirmation with details.')) ?>
+            </p>
+        <?php else : ?>
+            <p>
+                <?= $block->escapeHtml(__('We\'ll email you an order confirmation with details and tracking info.')) ?>
+            </p>
+        <?php endif; ?>
+    <?php endif; ?>
+
+    <?= $block->getAdditionalInfoHtml() ?>
+
+    <div class="actions-toolbar">
+        <div class="primary">
+            <a
+                class="
+                    button
+                    link--button
+                    success-page__button
+                    margin-vc-l
```

Comment:
what do you think about moving margin to `success-page__button` styles (same thing in `Magento_Checkout/templates/success.phtml`)? I'm hesitant when it comes to using atomic classes on alpaca level because changing these styles require overriding templates in child projects

Reply:
changed